src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,123 @@
+package duke;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Parser {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+
+    /**
+     * Reads the input given and returns the appropriate Command to execute.
+     *
+     * @param input User input string to parse.
+     * @return Command object.
+     * @throws DukeException If input is not recognised or does not follow format.
+     */
+    public static Command parse(String input) throws DukeException {
+        String[] wordArray = input.split(" ");
```

Comment:
Maybe you can consider naming this as `charArray`? I am not too sure, you can correct me if I'm wrong.